packages/yourloops/pages/profile/profile.tsx

```diff
@@ -0,0 +1,401 @@
+              <HomeIcon className={classes.homeIcon} />
+              {t("My Patients")}
+            </Link>
+          </Breadcrumbs>
+        </Toolbar>
+      </AppBar>
+      <Container className={classes.container} maxWidth="md">
+        <div style={{ display: "flex", flexDirection: "column", margin: "16px" }}>
+          <div className={classes.title}>{t("account-preferences-title")}</div>
+          <TextField
+            id="firstName"
+            value={firstName}
+            onChange={handleChange(setFirstName)}
+            error={errors.firstName}
+            helperText={errors.firstName && t("field-required")}
+            inputProps={{ className: classes.inputProps }}
+            InputProps={{
+              startAdornment: (
+                <InputAdornment className={classes.inputTitle} position="start">
+                  <span>{t("First name")}</span>
+                </InputAdornment>
+              ),
+            }}
+          />
+          <TextField
+            id="lastName"
+            value={name}
+            onChange={handleChange(setName)}
+            error={errors.name}
+            helperText={errors.name && t("field-required")}
+            inputProps={{ className: classes.inputProps }}
+            InputProps={{
+              startAdornment: (
+                <InputAdornment className={classes.inputTitle} position="start">
+                  <span>{t("Last name")}</span>
+                </InputAdornment>
+              ),
+            }}
+          />
+
+          {role === Roles.clinic ? (
+            <Fragment>
+              <TextField
+                id="mail"
+                value={mail}
+                disabled
+                onChange={handleChange(setMail)}
+                error={errors.mail}
+                helperText={errors.mail && t("Invalid email address.")}
+                className={classes.textField}
+                inputProps={{ className: classes.inputProps }}
+                InputProps={{
+                  startAdornment: (
+                    <InputAdornment position="start">
+                      <span>{t("Email")}</span>
+                    </InputAdornment>
+                  ),
+                }}
+              />
+              <Password
+                id="password"
+                label="password"
+                value={password}
+                error={errors.password}
+                helperText={t("password-too-weak")}
+                setState={setPassword}
+              />
+              <Password
+                id="passwordConfirmation"
+                label="confirm password"
+                value={passwordConfirmation}
+                error={errors.passwordConfirmation}
+                helperText={t("not-matching-password")}
+                setState={setPasswordConfirmation}
+              />
+            </Fragment>
+          ) : (
+            <Fragment>
+              <TextField
```

Comment:
Should we use a calendar pop-up for the bithdate?
Ask @MarineDBL about that ?
Also, at minimum, having the date format to be localized, is needed.

Reply:
She said "nope", but yes we need the date format to be localized